<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task</class>
 <widget class="QDialog" name="task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-30</y>
     <width>471</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QWidget" name="addTaskPage">
    <widget class="QDateEdit" name="dateEdit_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>290</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
    <widget class="QLabel" name="titleLabel_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>TITLE</string>
     </property>
    </widget>
    <widget class="QLabel" name="backgroundLabel_2">
     <property name="geometry">
      <rect>
       <x>-160</x>
       <y>20</y>
       <width>1061</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/Resources/logInBackground.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="descriptionLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>410</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="doneLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>IS DONE?</string>
     </property>
    </widget>
    <widget class="QLabel" name="addTaskLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>ADD NEW TASK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmTaskButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLabel" name="descriptionLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>DESCRIPTION</string>
     </property>
    </widget>
    <widget class="QLabel" name="importanceLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>IMPORTANCE</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="importanceSpinBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="titleLineEdit_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="doneComboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
    <zorder>backgroundLabel_2</zorder>
    <zorder>dateEdit_3</zorder>
    <zorder>dateLabel_2</zorder>
    <zorder>titleLabel_4</zorder>
    <zorder>descriptionLineEdit</zorder>
    <zorder>doneLabel</zorder>
    <zorder>addTaskLabel</zorder>
    <zorder>confirmTaskButton</zorder>
    <zorder>descriptionLabel</zorder>
    <zorder>importanceLabel</zorder>
    <zorder>importanceSpinBox</zorder>
    <zorder>titleLineEdit_4</zorder>
    <zorder>doneComboBox</zorder>
   </widget>
   <widget class="QWidget" name="editTaskPage">
    <widget class="QLineEdit" name="titleLineEdit_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="importanceLabel_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>IMPORTANCE</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>290</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="importanceSpinBox_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="doneComboBox_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="titleLabel_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>TITLE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="descriptionLineEdit_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>410</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
    <widget class="QLabel" name="descriptionLabel_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>DESCRIPTION</string>
     </property>
    </widget>
    <widget class="QLabel" name="doneLabel_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>IS DONE?</string>
     </property>
    </widget>
    <widget class="QLabel" name="backgroundLabel_4">
     <property name="geometry">
      <rect>
       <x>-160</x>
       <y>20</y>
       <width>1061</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/Resources/logInBackground.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmTaskButton_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLabel" name="addTaskLabel_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>EDIT TASK</string>
     </property>
    </widget>
    <zorder>backgroundLabel_4</zorder>
    <zorder>titleLineEdit_5</zorder>
    <zorder>importanceLabel_2</zorder>
    <zorder>dateEdit_4</zorder>
    <zorder>importanceSpinBox_2</zorder>
    <zorder>doneComboBox_2</zorder>
    <zorder>titleLabel_5</zorder>
    <zorder>descriptionLineEdit_2</zorder>
    <zorder>dateLabel_4</zorder>
    <zorder>descriptionLabel_2</zorder>
    <zorder>doneLabel_2</zorder>
    <zorder>confirmTaskButton_2</zorder>
    <zorder>addTaskLabel_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
